source/reviews.txt

```diff
@@ -0,0 +1,21 @@
+=================
+Code Review Guide
+=================
+
+This guide adapts the `Google Engineering Practices Documentation <https://github.com/google/eng-practices>`__
+
+The pages in this guide contain information on how you are expected to conduct
+reviews.
+
+.. cssclass:: hidden
+
+   .. toctree::
+      :titlesonly:
+
+      /reviews/standard
+      /reviews/speed
+      /reviews/comments
+      /reviews/pushback
+      /reviews/respect
+
```

Comment:
Not sure where to leave this comment, but do we want to define what requires a review and what doesn't? On the Cloud team, typo fixes and other straightforward changes (at the discretion of the author) don't require review.